source/documentation/runbooks/008-git-repository-approach.html.md.erb

```diff
@@ -0,0 +1,29 @@
+---
+owner_slack: "#data-catalogue"
+title: 008 Use single github repositry for Find MOJ data
+last_reviewed_on: 2024-02-26
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+## Status
+
+✅ Accepted
+
+## Context
+
+There was a discussion around what is the best git repo stategy to accomodate the following:
+ - Find MOJ Data App
+ - Python Datahub library
+ - Data registration API (still being discussed)
+
+ [This diagram](../../images/repo-strategy-workflow.excalidraw.png) highlights the differences between using one repo per component or a single repo for all
+
+## Decision
+
+The following was agreed:
```

Comment:
Can we add a distinct section for "Consequences", or list advantages/disadvantages as in the previous ADRs?

E.g. the main advantage is going to be ease of testing & reviewing changes that cross both layers of the app. And we're accepting that the build could get slower/more complicated

Might be worth also adding an initial ADR to introduce the format, e.g. https://github.com/nielsutrecht/adr-example/blob/initial/doc/adr/0001-record-architecture-decisions.md